tests/tck/features/steps/Steps.py

```diff
@@ -292,4 +292,10 @@ def step_impl(context):
 def step_impl(context):
     global exception
     assert exception != None
-    assert "Float overflow" in str(exception)
\ No newline at end of file
+    assert "Float overflow" in str(exception)
+
+@then(u'a SyntaxError should be raised at compile time: InvalidDelete')
+def step_imp(context):
+    global exception
+    assert exception != None
+    assert "DELETE can only be called on nodes and relationships" in str(exception)
```

Comment:
You can also delete paths
match p = ()-[]-() delete p

Reply:
This is (part of) the message currently prompted, so we need to change it as well if so.

Reply:
I guess we should.
Neo4j:
> Type mismatch: expected Node, Path or Relationship but was ...